artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/Transfer.java

```diff
@@ -43,7 +43,7 @@ public class Transfer extends InputAbstract {
    protected String sourceURL = DEFAULT_BROKER_URL;
 
    @Option(name = "--source-acceptor", description = "Acceptor used to build URL towards the broker. Default: 'artemis'.")
-   protected String sourceAcceptor;
+   protected String sourceAcceptor = "artemis";
```

Comment:
The `DEFAULT_BROKER_ACCEPTOR` could be reused here:
```suggestion
   protected String sourceAcceptor = DEFAULT_BROKER_ACCEPTOR;
```